packages/manager/modules/pci/src/projects/project/data-processing/submit-job/translations/Messages_fr_FR.json

```diff
@@ -10,5 +10,22 @@
   "data_processing_submit_job_stepper_submit_error_title": "Une erreur est survenue",
   "data_processing_submit_job_button_label": "Démarrer le job",
   "data_processing_submit_job_spinner_label": "lancement en cours…",
-  "data_processing_submit_job_error_label": "Une erreur est survenue, nous ne pouvons pas démarrer votre job pour le moment."
+  "data_processing_submit_job_error_label": "Une erreur est survenue, nous ne pouvons pas démarrer votre job pour le moment.",
+  "Client::UnprocessableEntity::odp.job.engine.unsupportedValue": "Erreur de saisie : le type de job demandé n'est pas supporté",
+  "Client::UnprocessableEntity::odp.job.engineVersion.unsupportedValue": "Erreur de saisie : la version de moteur demandée n'est pas supportée",
+  "Client::UnprocessableEntity::odp.job.region.unsupportedRegion": "Erreur de saisie : la région demandée n'est pas supportée",
+  "Client::BadRequest::odp.job.main_class_name.fieldMissing": "Erreur de saisie : la classe Main de votre job n'est pas renseignée",
+  "Client::BadRequest::odp.job.executorMemory.tooBig": "Erreur de saisie : la mémoire demandée pour les workers est trop importante",
+  "Client::BadRequest::odp.job.executorMemory.tooSmall": "Erreur de saisie : la mémoire demandée pour les workers est trop petite",
+  "Client::BadRequest::odp.job.driverMemory.tooBig": "Erreur de saisie : la mémoire demandée pour le driver est trop importante",
+  "Client::BadRequest::odp.job.driverMemory.tooSmall": "Erreur de saisie : la mémoire demandée pour le driver est trop petite",
+  "Client::BadRequest::odp.job.executorMemoryOverhead.tooBig": "Erreur de saisie : l'overhead mémoire demandé pour les workers est trop important",
+  "Client::BadRequest::odp.job.executorMemoryOverhead.tooSmall": "Erreur de saisie : l'overhead mémoire demandé pour les workers est trop petit",
+  "Client::BadRequest::odp.job.driverMemoryOverhead.tooBig": "Erreur de saisie : l'overhead mémoire demandé pour le driver est trop important",
+  "Client::BadRequest::odp.job.driverMemoryOverhead.tooSmall": "Erreur de saisie : l'overhead mémoire demandé pour le driver est trop petit",
+  "Client::BadRequest::odp.job.totalExecutorMemory.tooBig": "Erreur de saisie : la somme de la mémoire et de l'overhead demandés pour les workers est trop importante",
+  "Client::BadRequest::odp.job.totalDriverMemory.tooBig": "Erreur de saisie : la somme de la mémoire et de l'overhead demandés pour le driver est trop importante",
+  "Client::BadRequest::odp.project.notEnoughQuotas": "Erreur de quotas : les quotas de votre projet Public Cloud sont insuffisants. Veuillez consulter la documentation suivante pour les augmenter : <a class=\"oui-link oui-link_icon\" href=\"https://docs.ovh.com/fr/public-cloud/augmenter-le-quota-public-cloud/\" target=\"_blank\" rel=\"noopener\">Augmenter le quota Public Cloud<span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
```

Comment:
We recently added a state for Increasing Quota so maybe you can refer to this directly
in order to ease the user experience.
See: https://github.com/ovh/manager/pull/2664/files

Reply:
Commit amended with correction in translation, which redirects to the quotas increase page directly